nin/backend/compile.js

```diff
@@ -43,12 +47,34 @@ function res(projectPath, callback) {
     root += '/';
     root = root.replace(/\/\//g, '/');
 
-    const file = fs.readFileSync(root + stat.name);
+    let file = fs.readFileSync(root + stat.name);
```

Comment:
I changed it for a first implementation that redeclared `file`, but then I forgot to change it back when I changed to a different implementation that didn't redeclare it. Fixed.